selfdrive/controls/controlsd.py

```diff
@@ -376,7 +376,7 @@ def update_events(self, CS):
       self.events.add(EventName.posenetInvalid)
     if not self.sm['liveLocationKalman'].deviceStable:
       self.events.add(EventName.deviceFalling)
-    if not self.sm['liveLocationKalman'].inputsOK:
+    if not self.sm['liveLocationKalman'].inputsOK and not self.CP.notCar:
```

Comment:
we only want to do this when the body is controlled with a joystick, otherwise it relies on locationd

Reply:
disabled other locationd/paramsd/planner alerts too